src/screens/Settings/Experimental/FeatureInteger.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+
+import React, { useCallback, useState, useEffect } from "react";
+import { Switch, TextInput, StyleSheet } from "react-native";
+import { getEnvDefault } from "@ledgerhq/live-common/lib/env";
+
+import { View } from "react-native-animatable";
+import Track from "../../../analytics/Track";
+import colors from "../../../colors";
+
+type Props = {
+  name: *,
+  isDefault: boolean,
+  readOnly: boolean,
+  onChange: (name: string, val: mixed) => boolean,
+  value: number,
+  minValue: number,
+  maxValue: number,
+};
+
+const FeatureInteger = ({
+  name,
+  isDefault,
+  readOnly,
+  onChange,
+  value,
+  minValue,
+  maxValue,
+}: Props) => {
+  const constraintValue = useCallback(
+    v => {
+      let value = v;
+      if (typeof maxValue === "number" && parseInt(value, 10) > maxValue) {
+        value = maxValue;
+      }
+      if (typeof minValue === "number" && parseInt(value, 10) < minValue) {
+        value = minValue;
+      }
+
+      if (!value) {
+        value = maxValue || minValue;
```

Comment:
I don't really understand what this is doing. isn't the code above already doing this?

Reply:
unfortunaly no, when I didn't add that line, I would sometimes end up with a `undefined` inside the input